pkg/policycache/cache_test.go

```diff
@@ -145,6 +145,23 @@ func Test_Add_Remove(t *testing.T) {
 	}
 }
 
+func Test_Add_Remove_Any(t *testing.T) {
+	pCache := newPolicyCache(log.Log, dummyLister{}, dummyNsLister{})
+	policy := newAnyPolicy(t)
+	kind := "Pod"
+	pCache.Add(policy)
+	validateEnforce := pCache.get(ValidateEnforce, kind, "")
+	if len(validateEnforce) != 1 {
+		t.Errorf("expected 1 validate enforce policy, found %v", len(validateEnforce))
+	}
```

Comment:
we can add length check for for  mutate and generate type also.

Reply:
Added, PTAL.